helm/kong-app/templates/_helpers.tpl

```diff
@@ -25,13 +25,22 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{- define "kong.imagesHash" -}}
+{{- if .Values.image.registry }}
```

Comment:
Not sure, maybe?
```suggestion
{{- if .Values.image.registry -}}
```

Reply:
Tried locally, does not make any difference.